homeassistant/components/shopping_list/__init__.py

```diff
@@ -88,6 +97,18 @@ async def add_item_service(call: ServiceCall) -> None:
         if (name := call.data.get(ATTR_NAME)) is not None:
             await data.async_add(name)
 
+    async def remove_item_service(call: ServiceCall) -> None:
+        """Remove an item with `name`."""
```

Comment:
Would it make sense to improve the docstring to clarify the first item with a matching name will be removed? If yes, we should do that for the other services too.